daemon/web/index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="app" class="app"></div>
```

Comment:
Could we remove the need for `class="app"`? and use the id instead?

Reply:
Sass-lint prohibits use of ID selectors, and since I could only find one instance of its usage I opted to just change this to a class instead rather than disable the rule.

Reply:
hmm... ok....

Reply:
I suppose it's to discourage use of styles that only apply to a single element, which kind of makes sense